marc.py

```diff
@@ -555,10 +555,10 @@ def __init__(self, _db, library, integration):
         
     @classmethod
     def get_storage_settings(cls, _db):
-        integrations = ExternalIntegration.get_integrations_of_goal(
+        integrations = ExternalIntegration.for_goal(
             _db, ExternalIntegration.STORAGE_GOAL
         )
-        for integration in integrations:
+        for integration in integrations.all():
```

Comment:
This works but you don't need to call `all`. Iterating over the query object will run the query.